### ISSUE TITLE
[Historical MA] Data is missing/incorrect for 5/27-5/28 in states_daily_4pm_et.csv

### ISSUE THREAD

**Issue by andrewghaly** [COVID19Tracking/issues#470]
Data is missing for 5/27 and incorrect for 5/28

https://www.mass.gov/doc/covid-19-dashboard-may-27-2020/download - May 27 data

https://www.mass.gov/doc/covid-19-dashboard-may-28-2020/download - May 28 data

![image](https://user-images.githubusercontent.com/8738663/83368196-f1260500-a385-11ea-86ce-64f76a1c9c19.png)

**Comment by andrewghaly** [comment]
![image](https://user-images.githubusercontent.com/8738663/83369209-92628a80-a389-11ea-81db-4cc4b02e2c5c.png)

Please review

**Comment by muamichali** [comment]
Hi Andrew! Thanks. We will look into it ASAP.

**Comment by karaschechtman** [comment]
Hi @andrewghaly,
Thanks for bringing this to our attention! It looks like we updated our data before MA posted its update on 5/27, and instead carried over the numbers from 5/26. I have adjusted all the numbers to reflect to the PDF report on 5/27. 
However, I double checked everything on 5/28 and it all looked correct to me, so I left our data as is.

Before: 
![Screen Shot 2020-06-03 at 10 21 34 AM](https://user-images.githubusercontent.com/15933271/83648685-56474980-a584-11ea-9623-0aefdd689451.png)

After:
![Screen Shot 2020-06-03 at 10 26 16 AM](https://user-images.githubusercontent.com/15933271/83648990-b50cc300-a584-11ea-9524-564ec461164e.png)
